Benchmarks/Package.swift

```diff
@@ -0,0 +1,26 @@
+// swift-tools-version:5.10
```

Comment:
Should we run the benchmarks with at least Swift 6 (even if we need to run it in Swift 5 language mode)?

Reply:
Oh, that was just 'cause I copy-pasted from the JWTKit version of the file, let me fix it